vhdbuilder/packer/install-dependencies.sh

```diff
@@ -496,6 +496,66 @@ ls -ltr /usr/local/bin/* >> ${VHD_LOGS_FILEPATH}
 # shellcheck disable=SC2010
 ls -ltr /dev/* | grep sgx >>  ${VHD_LOGS_FILEPATH} 
 
+# populate the customized logrotate conf file for syslogs
+cat << EOF >> /etc/logrotate.d/aks-rsyslog
+/var/log/syslog
+{
+  rotate 4
+  daily
+  size 100M
+  missingok
+  notifempty
+  delaycompress
+  compress
+  postrotate
+      /usr/lib/rsyslog/rsyslog-rotate
+  endscript
+}
+
+/var/log/mail.info
+/var/log/mail.warn
+/var/log/mail.err
+/var/log/mail.log
+/var/log/daemon.log
+/var/log/kern.log
+/var/log/auth.log
+/var/log/user.log
+/var/log/lpr.log
+/var/log/cron.log
+/var/log/debug
+/var/log/messages
+/var/log/warn
+{
+  rotate 4
+  daily
+  size 25M
```

Comment:
did we determine this plays nicely with daily vs the (poorly documented) `maxsize`?

Reply:
Yep, from the manual testing I've done the size constraint is mutually exclusive from the daily - so if it does grow too large before the daily time frame is hit it will be rotated regardless.

Reply:
ended up switching to maxsize :)